CHANGELOG.md

```diff
@@ -21,6 +21,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
     - **bucket**
     - **key**
   - The `getObjectSize` function will no longer retry if the object does not exist
+- **CUMULUS-1930**
+  - The `@cumulus/common/util.uuid()` function has been removed
```

Comment:
Why isn't `setErrorStack` also listed here?

Reply:
`setErrorStack` had been previously deprecated, so removing it isn't considered a breaking change.